www/src/examples/Alert/Dismissible.js

```diff
@@ -1,28 +1,19 @@
-class AlertDismissibleExample extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      show: true,
-    };
-  }
+function AlertDismissibleExample() {
+  const [show, setShow] = React.useState(true);
```

Comment:
Should we modify our setup such that e.g. `useState` can be called directly by name, rather than as `React.useState`? The former convention seems more common.